bin/cqlsh.py

```diff
@@ -1865,8 +1883,9 @@ def do_login(self, parsed):
 
         LOGIN <username> (<password>)
 
-           Login using the specified username. If password is specified, it will be used
-           otherwise, you will be prompted to enter.
+           Login using the specified username. 
+           If password is specified it should be wrapped with single quotes.
```

Comment:
I have decided to change help here as I lost 30 minutes trying to make it working